go/vt/vtgate/planbuilder/operators/logical.go

```diff
@@ -233,6 +238,276 @@ func createOperatorFromDelete(ctx *plancontext.PlanningContext, deleteStmt *sqlp
 	return subq, nil
 }
 
+func createOperatorFromInsert(ctx *plancontext.PlanningContext, ins *sqlparser.Insert) (ops.Operator, error) {
```

Comment:
nit: `Insert` is an overloaded term here. Maybe name it `createInsertOpFromAST`? not v imporant, just sayin